Form/Editor/FormInspector.cs

```diff
@@ -0,0 +1,59 @@
+﻿using UnityEditor;
+using UnityEditorInternal;
```

Comment:
Fancy. What's this?

Reply:
It's an internal namespace used for all sorts of Editor-related stuff. He's using it for the ReorderableList... The downsides to this are it's not really documented and might be changed / removed in further versions without notice.